axiom/util_strings.c

```diff
@@ -423,8 +423,9 @@ int nr_str_char_count(const char* s, char c) {
   return count;
 }
 
-char* nr_str_append(char* dest, const char* src) {
+char* nr_str_append(char* dest, const char* src, const char* delimiter) {
   char* tmp = NULL;
+  const char* delim = delimiter? delimiter : ""; // Treat NULL delimiter as no delimiter
```

Comment:
```suggestion
  const char* delim = delimiter ? delimiter : ""; // Treat NULL delimiter as no delimiter
```

Reply:
This line needs to be clang formatted.

Reply:
Maybe clearer to make test (NULL == delimiter)?

Reply:
Done with https://github.com/newrelic/newrelic-php-agent/pull/467/commits/5f8f238154b84e281b3303511f194cbda6881ef2